### PR TITLE
Suppress error message from grep when checking whether a subnet has a secondary range or not.

### PR DESCRIPTION
**Pull request by jingax10**
**What this PR does / why we need it**:

Get rid of stdrr caused by grep command when running cluster/kube-up.sh for GCE.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

N/A

**Special notes for your reviewer**:

No behavior change.

**Release note**:

```release-note
"NONE"
```

### CODE REVIEW

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -1713,7 +1713,7 @@ function create-subnetworks() {
       --secondary-range "services-default=${SERVICE_CLUSTER_IP_RANGE}"
     echo "Created subnetwork ${IP_ALIAS_SUBNETWORK}"
   else
-    if ! echo ${subnet} | grep --quiet secondaryIpRanges ${subnet}; then
+    if ! echo ${subnet} | grep -qs secondaryIpRanges ${subnet} 2>/dev/null; then
```
Comment (bowei) [inline cluster/gce/util.sh]: Looks like the problem is with the second `${subnet}`:

`if ! echo ${subnet} | grep -qs secondaryIpRanges; then` should work

Comment (jingax10) [reply cluster/gce/util.sh]: You are right! Fixed it. PTAL

**Pull request merged**